src/e2e/java/teammates/e2e/cases/e2e/BaseE2ETestCase.java

```diff
@@ -146,4 +158,50 @@ protected AdminHomePage loginAdmin() {
         return loginAdminToPage(createUrl(Const.WebPageURIs.ADMIN_HOME_PAGE), AdminHomePage.class);
     }
 
+    /**
+     * Deletes file with fileName from the downloads folder.
+     */
+    protected void deleteDownloadsFile(String fileName) {
+        String filePath = getTestDownloadsFolder() + fileName;
+        FileHelper.deleteFile(filePath);
+    }
+
+    /**
+     * Compares file with fileName in the downloads folder with expected file in results folder.
+     */
+    protected void verifyDownloadedFileContent(String expectedFileName, String actualFileName) {
+        try {
+            String expectedFilePath = getTestResultsFolder() + expectedFileName;
+            String actualFilePath = getTestDownloadsFolder() + actualFileName;
+            assertEquals(FileHelper.readFile(expectedFilePath), FileHelper.readFile(actualFilePath));
```

Comment:
This works, but we don't really need to do a full check (otherwise we need to also maintain these test files). Just doing a quick sanity check (file name and maybe a few representative lines) is good enough.